LICENSE

```diff
@@ -1,7 +1,6 @@
 MIT License
 
-Copyright (c) 2019-2021 Camelot Developers
-Copyright (c) 2018-2019 Peeply Private Ltd (Singapore)
+Copyright © 2023 Jelmer Draaijer
```

Comment:
This is not OK. I will not accept the PR like this.

Reply:
Reverted